libr/io/p/io_mach.c

```diff
@@ -197,32 +188,30 @@ static ut64 getNextValid(RIO *io, RIODesc *fd, ut64 addr) {
 	return lower;
 }
 
-static int __read(RIO *io, RIODesc *desc, ut8 *buf, int len) {
+static int __read(RIO *io, RIODesc *fd, ut8 *buf, int len) {
```

Comment:
Fd != desc aka naming regression

Reply:
naming regression, srsly?

Reply:
I prefer single underscore. Double underscores are reserved for implementations.

http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf
All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use.